docs/CHANGES.md

```diff
@@ -4,6 +4,7 @@
 
 - #585 Duplicate contact created due to JID case sensivity [saganshul]
 - #628 Fixes the bug in displaying chat status during private chat. [saganshul]
+- #797 Time format made configurable, with a choice between 12hr and 24hr format. [smitbose]
```

Comment:
Update this entry please. Still falsely creates the impression that there's 2 choices.